dataset.html

```diff
@@ -130,11 +130,14 @@ <h2 id="readme">README</h2>
 
   </div><!-- / dataset show -->
 </div>
-
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
 <script src="vendor/d3/d3.min.js"></script>
 <script src="vendor/vega/vega.min.js"></script>
 <script src="vendor/vega-lite/vega-lite.min.js"></script>
 <script src="vendor/vega-embed/vega-embed.js" charset="utf-8"></script>
 
-<script src="bundle.js" charset="utf-8"></script>
+<script src="vendor/csv/csv.js"></script>
+<script src="vendor/handsontable/handsontable.full.js"></script>
```

Comment:
Can we get this working with webpack? We should not be putting stuff into the page other than bundle.js if we can help it.